docs(changelog): howto in editor page WDOC-322 (#859)
scaleway/docs-content: editor-components.mdx

---

## Log (Changelog)

🧩 &nbsp;This functionnality uses **MDX** files.

See [Changelog page](https://www.scaleway.com/en/docs/changelog/).

Each "Log" is MDX file:

> - Go to `changelog/` folder

Reviewer: ```suggestion
- Go to the `changelog/` folder
```

- Add or choose a directory relative to the log you want to add
- Create a `.mdx` file in it 

NOTES: 

> - Organize `changelog/` folder structure as you want

Reviewer: ```suggestion
- Organize the `changelog/` folder structure as you want
```

- The `.mdx` filename does not matter 

> - Assets should be in a `asset` directory, sibling of your `.mdx` file

Reviewer: ```suggestion
- Assets should be in an `asset` directory, sibling of your `.mdx` file
```

### Log file frontmatter

> At top of `.mdx` file, you MUST add data in frontmatter:

Reviewer: ```suggestion
At the top of ` the.mdx` file, you MUST add the frontmatter data:
```

```
---
title: Migration to the new S3 backend (HIVE) for all regions
status: changed
author:
    fullname: 'Join the #container-registry channel on Slack.'
    url: 'https://slack.scaleway.com/'
date: 2022-01-02
category: compute
product: container-registry
---
```

- **title** (mandatory)
- **status** (mandatory)
    - `added`
    - `security`
    - `changed`
    - `deprecated`
    - `removed`
    - `fixed`
- **author**

>     - **fullname** (optional / mandatory IF url is set) It could be a name or a sentence if you please.

Reviewer: ```suggestion
    - **fullname** (optional / mandatory IF url is set). It could be a name or a sentence, as you prefer
```

>     - **url** (optionnal)

Reviewer: ```suggestion
    - **url** (optional)
```

- **date** (mandatory) `YYYY-MM-DD`
- **category** (mandatory) `kebab-case`
- **product** (mandatory) `kebab-case`

### Log file contents

> The content section works as any documentation `.mdx` file. It accept markdown and Components as well. Be concise !

Reviewer: ```suggestion
The content section works as any documentation `.mdx` file. It accepts both markdown and components. Be concise!
```

#### About Download link in content

- Use `[Download](document-pdf-name.pdf) the document as PDF file.`

> - Place the PDF file at a sibling assets directory

Reviewer: ```suggestion
- Place the PDF file in a sibling assets directory
```

>     - This is usefull to automatically get the filesize data

Reviewer: ```suggestion
    - This is useful to automatically get the filesize data
```

> - Then it should appear like this (with PDF icon):

Reviewer: ```suggestion
- Then, it should appear like this (with PDF icon):
```
